www/source/partials/docs/_bp-iterative-development.html.md.erb

```diff
@@ -1,16 +1,24 @@
+<<<<<<< HEAD
```

Comment:
Merge conflict here as well and right below

Reply:
Ugh I thought I fixed these. Will try again.